From: Carlos Maddela <e7appew@gmail.com>
Date: Sun, 8 Jan 2017 23:27:27 +1100
Subject: Fix desktop files to use valid field code with Exec key.

Description: Fix desktop files to use valid field code with Exec key.
Author: Carlos Maddela <e7appew@gmail.com>
Origin: vendor
Forwarded: https://bugzilla.gnome.org/show_bug.cgi?id=790254
Last-Update: 2017-11-12
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 tool/seahorse-pgp-encrypted.desktop.in.in | 2 +-
 tool/seahorse-pgp-keys.desktop.in.in      | 2 +-
 tool/seahorse-pgp-signature.desktop.in.in | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/tool/seahorse-pgp-encrypted.desktop.in.in b/tool/seahorse-pgp-encrypted.desktop.in.in
index 2d631e8..0897aa0 100644
--- a/tool/seahorse-pgp-encrypted.desktop.in.in
+++ b/tool/seahorse-pgp-encrypted.desktop.in.in
@@ -1,7 +1,7 @@
 [Desktop Entry]
 _Name=Decrypt File
 MimeType=application/pgp-encrypted;
-Exec=seahorse-tool --decrypt
+Exec=seahorse-tool --decrypt %U
 Type=Application
 Terminal=false
 NoDisplay=true
diff --git a/tool/seahorse-pgp-keys.desktop.in.in b/tool/seahorse-pgp-keys.desktop.in.in
index 36cb153..5b4bd73 100644
--- a/tool/seahorse-pgp-keys.desktop.in.in
+++ b/tool/seahorse-pgp-keys.desktop.in.in
@@ -1,7 +1,7 @@
 [Desktop Entry]
 _Name=Import Key
 MimeType=application/pgp-keys;
-Exec=seahorse-tool --import
+Exec=seahorse-tool --import %U
 Type=Application
 Terminal=false
 NoDisplay=true
diff --git a/tool/seahorse-pgp-signature.desktop.in.in b/tool/seahorse-pgp-signature.desktop.in.in
index 00b9562..77820f7 100644
--- a/tool/seahorse-pgp-signature.desktop.in.in
+++ b/tool/seahorse-pgp-signature.desktop.in.in
@@ -1,7 +1,7 @@
 [Desktop Entry]
 _Name=Verify Signature
 MimeType=application/pgp-signature;
-Exec=seahorse-tool --verify
+Exec=seahorse-tool --verify %U
 Type=Application
 Terminal=false
 NoDisplay=true
